internal/devicescale/impl_unix.go

```diff
@@ -37,6 +38,12 @@ const (
 	desktopXfce
 )
 
+var (
+	cachedScale    float64
+	cachedAt       int64
+	scaleExpiresMS = int64(16)
```

Comment:
How about using nano time? The code would be:

```
scaleExpires = time.Second / 60

// ...

now := time.Now().UnixNano()
if now-cachedAt < scaleExpiresMS {

// ...

cachedAt = now // now can be reused here: I don't think now should be updated.
```

Wdyt?

Reply:
Implementing this now.

Reply:
Oh also, I recalculated 'now' because of how long gsettings was taken.

Reply:
OK so the time how long gsettings is taken cannot be ignored. Thanks!